drivers/census_tracking/app/views/census_tracking/warehouse_reports/census_trackers/index.haml

```diff
@@ -0,0 +1,10 @@
+- title = _('Census Tracking Worksheets')
+- content_for :title, title
+
+= render '/warehouse_reports/breadcrumbs', tab: 'operational'
+%h1.mb-5= title
+
+= render 'filter'
+
+- if @filter.project_ids.present? || @filter.project_group_ids.present? || @filter.organization_ids.present?
```

Comment:
If we're going to require these, we should throw an error (flash message probably) on submission if they are missing.

Reply:
We should call it out explicitly on the filter section too.